document/FileFormat.mdk

```diff
@@ -2672,16 +2675,20 @@ This can be used, for example, to designate the target of a jump instruction, or
 * `"declaration"`: The location introduces into the program a name which denotes an entity such as a variable, function, template, _etc_.
 
 * `"functionEnter"`: This location is an entry point to a function or method.
+    Every `annotatedCodeLocation` whose `kind` property is `"functionEnter"` shall be paired with a
+    subsequent `annotatedCodeLocation` whose `kind` property is `"functionExit"` and which refers to the same function,
+    unless the `codeFlow` in which the call occurs terminates before a function exit point is reached. 
 
-* `"functionExit"`: This location is an exit point from a function or method (for example, a return statement).
+* `"functionExit"`: This location represents the actual exit point from the function.
```

Comment:
Fixed. I put this description (except for the part about possibly being preceded by a `functionReturn`) in a NOTE, so as not to normatively state that these are the only possibilities.

---
In reply to: [77242760](https://github.com/sarif-standard/sarif-spec/pull/264#discussion_r77242760) [](ancestors = 77242760)